pkg/planner/core/tiflash_selection_late_materialization.go

```diff
@@ -285,6 +285,4 @@ func PushedDown(sel *PhysicalSelection, ts *PhysicalTableScan, selectedConds []e
 	removeSpecificExprsFromSelection(sel, selectedConds)
 	// add the pushed down conditions to table scan
 	ts.LateMaterializationFilterCondition = selectedConds
-	// Update the row count of table scan after pushing down the conditions.
-	ts.StatsInfo().RowCount *= selectedSelectivity
```

Comment:
We should not update the ```RowCount``` after the ```physicalOptimize```. when update ```stats``` in the ```physicalOptimize```, we can check it by the ```cost tracer```.